src/intersections/lineraysegmentplane.jl

```diff
@@ -0,0 +1,96 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+#=
+Return appropriate type for a geometry overlapping with a `Plane`.
+Ideally this would be a macro, but the geometry type isn't known at parse time, so it is implemented via multiple-dispatch instead
```

Comment:
```suggestion
Ideally this would be a macro, but the geometry type isn't known at parse time,
so it is implemented via multiple-dispatch instead
```